src/components/LineChart.js

```diff
@@ -2,6 +2,7 @@
 import React from 'react';
 import { Select } from 'antd';
 import Highcharts from 'highcharts';
+import no_data_to_display from 'highcharts/modules/no-data-to-display.js';
```

Comment:
this is a nitpick - but i'd import it as noDataToDisplay :) to follow the CamelCase JS way.